### PR TITLE
Argo cd external link

### PR DESCRIPTION
**Pull request by oded-komodor**
Add a tutorial that demonstrates how to integrate ArgoCD external link to the Komodor
based on the Kubernetes workload resources annotation to extend Argo’s functionality

### CODE REVIEW

File: docs/Integrations/ArgoCD.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+3. Check the rendered output (headings, lists, code blocks, tables) for proper
+   formatting and use a linkchecker before you publish this page.
+
+Conversion notes:
+
+* Docs to Markdown version 1.0β33
+* Sat Aug 06 2022 12:09:45 GMT-0700 (PDT)
+* Source doc: ArgoCD - Komodor integration
+----->
+
```
Comment (amit9192) [inline docs/Integrations/ArgoCD.md:23]: @oded-komodor - Add title?

File: docs/Integrations/ArgoCD.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+This tutorial demonstrates how to integrate ArgoCD external link to the Komodor application for using Komodor as a centralized troubleshooting platform that allows users 
+
+gain effectively investigate Kubernetes issues providing a clear understanding of all events and historical context across a timeline. \
+ \
+ \
```
Comment (amit9192) [inline docs/Integrations/ArgoCD.md:29]: This extra \ is going to be presented on the page - you should probably drop it :)

File: docs/Integrations/ArgoCD.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+* Docs to Markdown version 1.0β33
+* Sat Aug 06 2022 12:09:45 GMT-0700 (PDT)
+* Source doc: ArgoCD - Komodor integration
+----->
+
+
+This tutorial demonstrates how to integrate ArgoCD external link to the Komodor application for using Komodor as a centralized troubleshooting platform that allows users 
+
+gain effectively investigate Kubernetes issues providing a clear understanding of all events and historical context across a timeline. \
```
Comment (amit9192) [inline docs/Integrations/ArgoCD.md:27]: Is that a new sentence or part of the previous one?

File: docs/Integrations/ArgoCD.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Conversion notes:
+
+* Docs to Markdown version 1.0β33
+* Sat Aug 06 2022 12:09:45 GMT-0700 (PDT)
+* Source doc: ArgoCD - Komodor integration
+----->
+
+
+This tutorial demonstrates how to integrate ArgoCD external link to the Komodor application for using Komodor as a centralized troubleshooting platform that allows users
```
Comment (amit9192) [inline docs/Integrations/ArgoCD.md:25]: Suggestion: This tutorial demonstrates how to utilize ArgoCD external links (+ make it a URL - https://argo-cd.readthedocs.io/en/stable/user-guide/external-url/) to link to the Komodor platform...

File: docs/Integrations/ArgoCD.md (hunk 3 above)
Comment (amit9192) [inline docs/Integrations/ArgoCD.md:27]: Komodor allows users to effectively investigate Kubernetes-related issues by providing a clear understanding of all events and...

File: docs/Integrations/ArgoCD.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+gain effectively investigate Kubernetes issues providing a clear understanding of all events and historical context across a timeline. \
+ \
+ \
+
+
+**General**
+
+ArgoCD allows adding an external link to the Komodor service based on the Kubernetes workload resources annotation to extend Argo’s functionality
+
+ \
```
Comment (amit9192) [inline docs/Integrations/ArgoCD.md:36]: You go down two lines here (see the preview)

**Review by amit9192 — CHANGES_REQUESTED** [review]

**Comment by oded-komodor** [comment]
closing PR and adding a new MD file

**Pull request closed without merging**